christmas/views.py

```diff
@@ -1,3 +1,49 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect
+from datetime import datetime
+from random import shuffle
 
-# Create your views here.
+def index(request):
+    today = datetime.now()
+    is_christmas = "Yes!" if today.month == 12 and today.day == 25 else "No!"
```

Comment:
make is_christmas to have a boolean value. And lets the word choice be part of the interface layer